tools/interp.cpp

```diff
@@ -37,7 +37,7 @@ int main(int argc, char** argv) {
       auto code = read_wasm( filename );
 
       // Instaniate a new backend using the wasm provided.
-      backend<std::nullptr_t, jit_profile, default_options, profile_instr_map> bkend( code, &wa );
+      backend<std::nullptr_t, interpreter, default_options, profile_instr_map> bkend( code, &wa );
```

Comment:
Either remove all the profiling related code or use interpreter+profile.

Reply:
The argument processing should be removed too.

Reply:
Sorry about that Steven. Trying to make these changes on my phone at the dentists office currently.